src/nls/de/strings.json

```diff
@@ -1,5 +1,491 @@
 {
+    "ACTIONS": {
+        "ADD_STROKE": "[8413887] ü_Add Stroke",
```

Comment:
Yes, once a new string is added, the process is:
1. ALF scans and harvests.
2. Use localization mock string instead English string, as shown in this line.
3. Send Lockit which contains all untranslated strings to linguists.
4. After translation comes back, ALF use real localized string substitutes the mock string.